ui/app/src/Utils/Types.elm

```diff
@@ -1,20 +1,19 @@
 module Utils.Types exposing (..)
 
-import Http
 import Time
 
 
-type ApiResponse e a
+type ApiData a
     = Initial
     | Loading
-    | Failure e
+    | Failure String
```

Comment:
We can always change the `Failure String` to be a `Failure` of our own union type if we need to extract additional information.